app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/landing/LandingViewModel.kt

```diff
@@ -191,7 +191,12 @@ class LandingViewModel @Inject constructor(
     }
 
     private fun handleCreateAccountClicked() {
-        sendEvent(LandingEvent.NavigateToCreateAccount)
+        // TODO PM-9401: ADD FEATURE FLAG email-verification
+        if (true) {
```

Comment:
I think we should either default this false or get the flag implemented before merging this to `main`

Reply:
Agree I'll remove the navigation and we implement it with the feature flag afterwards